explore/src/main/scala/explore/config/sequence/GmosSequenceTable.scala

```diff
@@ -72,14 +72,18 @@ sealed trait GmosSequenceTable[S, D]:
     currentVisitData.map(_._2)
   protected[sequence] lazy val currentAtomExecutedSteps: Option[Int]         = currentVisitData.map(_._3)
 
+  // Hide acquisition when science is executing.
+  protected[sequence] lazy val isAcquisitionDisplayed: Boolean =
+    !currentAtomSequenceType.contains_(SequenceType.Science)
```

Comment:
Right, `!exists(Science)` instead of `exists(Acquistion)`. The former returns `true` on `None` (which means the sequence hasn't started executing), while the latter returned `false`.